roiextractors/extractors/caiman/caimansegmentationextractor.py

```diff
@@ -84,7 +84,38 @@ def roi_locations(self):
 
     @staticmethod
     def write_segmentation(segmentation_object, savepath):
-        raise NotImplementedError
+        from scipy.sparse import csc_matrix
+        if savepath.split('.')[-1]!='hdf5':
+            raise ValueError('filetype to save must be *.hdf5')
+        with h5py.File(savepath,'w') as f:
+            #create base groups:
+            estimates = f.create_group('estimates')
+            params = f.create_group('params')
+            #adding to estimates:
+            if segmentation_object._roi_response_neuropil:
+                estimates.create_dataset('C',data=segmentation_object._roi_response_neuropil)
+            estimates.create_dataset('F_dff', data=segmentation_object._roi_response_fluorescence)
+            if segmentation_object._roi_response_deconvolved:
+                estimates.create_dataset('S', data=segmentation_object._roi_response_deconvolved)
+            if segmentation_object._images_mean:
```

Comment:
Cn is not a mean image

Reply:
fixed here https://github.com/catalystneuro/roiextractors/pull/34/commits/82f90e6fe7770dc07c6e3f9024e870986d8c123f